### DISCUSSION TITLE
Any Calibre users out there? Could yuse some help installing DeDrm into Calibre?

### DISCUSSION THREAD

**Discussion by FredNugent** [apprenticeharper/DeDRM_tools#2167] [Q&A]
Hi all,

Newb question here. Needless to say I'm not a programmer. I can install things if given instructions but that's about the limit of my nous!

I've downloaded DeDrm_tools-master. My problem is that Calibre (v 6.8) seems to only install zipped plugins. I find the DeDrm_plugin folder easily enough inside the zipped DeDrm_tools-master folder, but the DeDrm_plugin plugin folder isn't zipped and Calibre doesn't seem to know what to do with it. I've tried zipping the DeDrm-plugin folder and installing it that way but still I get an error message.

Clearly I'm missing something ~ probably something basic. I'd greatly appreciate any help.

FWIW I"m using Calibre 6.8 and my goal is to convert .azw files to epub so I can read on my Kobo.

Best,

Fred

**Comment by ElleKayEm** [comment]
Download the latest release DeDRM 10.0.3 from here: https://github.com/noDRM/DeDRM_tools/releases/tag/v10.0.3

You want the 1st link under assets.  Unzip DeDRM_tools_10.0.3.zip and load DeDRM_plugin.zip.

**Reply by ElleKayEm** [reply]
When you're ready to decrypt those Amazon books, take a look at this post:  https://www.mobileread.com/forums/showthread.php?t=283371

**Reply by FredNugent** [reply]
Thanks very much :) I got it installed at long last. It's not working but hopefully I'm headed the right direction.

**Comment by ElnoExistemas** [comment]
Got it to work on Calibre 6.11 today (1 February 2023) by reading a guide that I stumbled on in a web search.  Many steps, requiring careful reading, taking notes, and the information did not conform perfectly with the dialog text in Calibre.  Had to enter my Kindle serial number and add 2 plugins, customizing the "DeDRM_plugin.zip" file.  Felt lucky that it worked, have tested it simply by adding the Amazon ebook to Calibre from my downioads folder and sending via USB it to my preferred other unregistered Kindle, no need to convert the ebook, it is done automatically.  May be more straightforward with the Ubuntu that I use.
